terraform/stack_staging/main.tf

```
@@ -13,7 +13,7 @@ module "stack_staging" {
   domain_name      = local.staging_domain_name
   cert_domain_name = local.cert_domain_name
 
-  min_capacity = 0
+  min_capacity = 1
```

```suggestion
  min_capacity = 0
```

We let the service scale down when it's not in use.